pkg/logs/internal/tag/local_provider_test.go

```diff
@@ -37,6 +37,7 @@ func TestLocalProviderExpectedTags(t *testing.T) {
 
 	oldStartTime := coreConfig.StartTime
 	coreConfig.StartTime = clock.Now()
+	coreConfig.SetDetectedFeatures(coreConfig.FeatureMap{})
```

Comment:
I think this doesn't hurt?  There are a lot of issues with detected features and the order in which tests run, so this might fix one of them.

Reply:
I added this to fix an issue on my M1 where this test would fail regularly. The strange part of it was that it only happened when running from IDE and not when running from `invoke` 🤷🏽‍♂️

Reply:
I think this is because the IDE only runs one test function, and this test implicitly depends on other test functions having set up the detected features.